client/gutenberg/extensions/publicize/service-icon.jsx

```diff
@@ -0,0 +1,73 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { Icon } from '@wordpress/components';
+
+/**
+ * Module variables
+ */
+// @TODO: Import those from https://github.com/Automattic/social-logos when that's possible
```

Comment:
This could maybe provide little more context for future lookers.

i.e. why it isn't possible now?

Reply:
Sure 👍 https://github.com/Automattic/wp-calypso/pull/29317/commits/7d7cabab99d976360407891380d627bcdb1c5e0d